### ISSUE TITLE
Extented Form Widget duplicates first Row

### ISSUE THREAD

**Issue by DerThorsten** [fyne-io/fyne#694]
### Describe the bug:
When  I try to extend a Form widget and append rows to it, the first row is duplicated.

### To Reproduce:
See code below
### Screenshots:
This what I get from the code below:
![actual](https://user-images.githubusercontent.com/904752/75048164-5eddad80-54c8-11ea-89be-624c0f145b8a.png)

This what I would expect:
![Screenshot from 2020-02-21 16-36-14](https://user-images.githubusercontent.com/904752/75048287-93516980-54c8-11ea-8c44-cd7d2241ca40.png)

### Example code:
```go
package main

import (
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

type MyForm struct {
	widget.Form
}

func NewMyForm() *MyForm {

	name := widget.NewEntry()
	form := &MyForm{}
	form.ExtendBaseWidget(form)
	form.Append("Name", name)
	return form
}

func main() {
	a := app.New()
	w := a.NewWindow("Hello")

	w.SetContent(NewMyForm())

	w.ShowAndRun()
}

```

### Device
 - OS: Ubuntu 19.10
 - Go version: 1.13
 - Fyne version: 1.2.0

**Comment by andydotxyz** [comment]
Thanks, it seems we missed Form when the widget extending code was added...

**Comment by andydotxyz** [comment]
Resolved on the releases/v1.2.3 branch, release pending
